app/views/application/_navbar.html.erb

```diff
@@ -1,9 +1,32 @@
-<nav>
-<h3><%= link_to t("nav.home"), root_path %> |
-<% if current_user %>
-  <%= link_to t("nav.signout"), session_path(current_user), method: :delete %> |
-<% else %>
-  <%= link_to t("nav.signup"), new_users_path %> |
-  <%= link_to t("nav.signin"), new_session_path %></h3>
-<% end %>
-</nav>
+<div class="navbar">
+    <% if current_user %>
```

Comment:
Is the indentation correct in this file? seems like more than 2 spaces...

Reply:
No, sorry, Done in sha: a2fa585